### ISSUE TITLE
L-moments, FDCs, and DVS 

### ISSUE THREAD

**Issue by scworland-usgs** [scworland-usgs/restore-2018#6]
We had originally thought @wasquith-usgs would explore direct predictions of streamflow statistics, and @scworland-usgs would explore estimating daily values. I believe we are converging on a combined flow duration curve (FDC) approach for both methods. There are computational and theoretical advantages to using the same approach for both methods. Here is a 30 ft view of how I see the major tasks:

1. Use mobile basin (HU4=0316) for developing methods
2. Calculate L-moments for all (n=67) gages in mobile basin
3. Use L-moments to approximate FDCs
4. Regionalize L-moments 
5. Quantify alteration at each site (gaged and ungaged) following the literature
6. Use time series of EPs from donor sites to generate dvs at gaged sites
7. Scale up to RESTORE footprint.

Steps 3-6 will require extensive exploration and research–the details of which should be handled in the "predict FDC" project page. I propose that @wasquith-usgs takes the lead on steps 2 and 3, @scworland-usgs will lead steps 4 and 6, and @rrknight-usgs leads 5. We can think about 7 later. Any objections? Of course "taking the lead" doesn't preclude input from others, but I want to ensure we are poised to push this thing over the finish line in time!

**Comment by wasquith-usgs** [comment]
One perspective that is a bit missing is whether or how or when we might regionalize dimensionless forms of the FDC. Regionalizing the L-skew + is sort of that already because higher moments are dimensionless. However, L-scale will continue to have units of cfs or log10(cfs).  The precipitation frequency world and homogenous regionalization of L-moments in many settings remove dimension first on the data (usually divide by sample mean) and then dump out tables of L-moments with L-scale = coefficient of L-variation and regionalize the dimensionless frequency curve.

**Comment by scworland-usgs** [comment]
@wasquith-usgs I agree with you that it is something we need to discuss but my hope was that this particular thread could stay further zoomed out (i.e., focus on big picture goals and then handle the details in subsequent issues or within the [project page](https://github.com/scworland-usgs/restore-2018/projects/4)). I have a tendency to get bogged down in details that are supporting an overall objective and am trying to at least help myself stay on course. I started a new issue for dimensions here #7.

**Comment by scworland-usgs** [comment]
After we agree to these tasks I will add them to out project management page so we can be sure we are moving forward.

@wasquith-usgs 
1. Explore mixtures of distributions and develop function that takes L-moments and returns fit that can be used to move between probability and quantiles.
2. Finalize a logistic regression using full covariate dataset and update probability of zero flows in the fdc_lmr_pplo.feather file.
3. Continue soap-film/GAM approaches.

@scworland-usgs
1. Finish bare-bones script to calculate DVs using donor sites.
2. Explore machine-learning regression models
3. Explore bayesian models

@rrknight-usgs 
1. Distill methods for calculating altered and unaltered FDCs
2. Keep the train on the tracks
